api/core/rag/datasource/vdb/elasticsearch/elasticsearch_vector.py

```diff
@@ -37,12 +43,19 @@ class ElasticSearchVector(BaseVector):
     def __init__(self, index_name: str, config: ElasticSearchConfig, attributes: list):
         super().__init__(index_name.lower())
         self._client = self._init_client(config)
+        self._version = self._get_version()
+        self._check_version()
```

Comment:
The `_version` is used only once. Consider nest the `_get_version` method inside `_check_version` without holding it as member variable of each ElasticSearchVector instance.